contrib/toxic/patches/makefile.patch

```diff
@@ -0,0 +1,39 @@
+--- a/Makefile
++++ b/Makefile
+@@ -5,11 +5,11 @@
+ 
+ LIBS = toxcore ncursesw libconfig libcurl
+ 
+-CFLAGS ?= -std=c11 -pthread -Wall -Wpedantic -Wunused -fstack-protector-all -Wvla -Wno-missing-braces -Wmissing-prototypes -Wcast-align -Wcast-qual -Wmissing-declarations -Wshadow -Wunused-macros -Wformat-nonliteral -Wno-unknown-pragmas
++CFLAGS = -std=gnu11 -pthread -Wall -Wpedantic -Wunused -fstack-protector-all -Wvla -Wno-missing-braces -Wmissing-prototypes -Wcast-align -Wcast-qual -Wmissing-declarations -Wshadow -Wunused-macros -Wformat-nonliteral -Wno-unknown-pragmas
+ CFLAGS += '-DTOXICVER="$(VERSION)"' -DHAVE_WIDECHAR -D_XOPEN_SOURCE_EXTENDED -D_FILE_OFFSET_BITS=64
+ CFLAGS += '-DPACKAGE_DATADIR="$(abspath $(DATADIR))"'
+ CFLAGS += ${USER_CFLAGS}
+-LDFLAGS ?=
++LDFLAGS =
+ LDFLAGS += ${USER_LDFLAGS}
+ 
+ OBJ = autocomplete.o avatars.o bootstrap.o chat.o chat_commands.o conference.o configdir.o curl_util.o execute.o
+@@ -72,19 +72,16 @@
+ all: $(BUILD_DIR)/toxic
+ 
+ $(BUILD_DIR)/toxic: $(OBJ)
+-	@echo "  LD    $(@:$(BUILD_DIR)/%=%)"
+-	@$(CC) $(CFLAGS) -o $(BUILD_DIR)/toxic $(OBJ) $(LDFLAGS)
++	$(CC) $(CFLAGS) -o $(BUILD_DIR)/toxic $(OBJ) $(LDFLAGS)
+ 
+ $(BUILD_DIR)/osx_video.o: $(SRC_DIR)/$(OSX_VIDEO)
+-	@echo "  CC    $(@:$(BUILD_DIR)/)osx_video.o"
+-	@$(CC) $(CFLAGS) -o $(BUILD_DIR)/osx_video.o -c $(SRC_DIR)/$(OSX_VIDEO)
++	$(CC) $(CFLAGS) -o $(BUILD_DIR)/osx_video.o -c $(SRC_DIR)/$(OSX_VIDEO)
+ 
+ $(BUILD_DIR)/%.o: $(SRC_DIR)/%.c
+ 	@if [ ! -e $(BUILD_DIR) ]; then \
+ 		mkdir -p $(BUILD_DIR) ;\
+ 	fi
+-	@echo "  CC    $(@:$(BUILD_DIR)/%=%)"
+-	@$(CC) $(CFLAGS) -o $(BUILD_DIR)/$*.o -c $(SRC_DIR)/$*.c
++	$(CC) $(CFLAGS) -o $(BUILD_DIR)/$*.o -c $(SRC_DIR)/$*.c
```

Comment:
these also do nothing at all, what is this patch meant to do? it doesn't fix anything